provider/aws/dist/rack.json

```diff
@@ -1337,7 +1337,7 @@
             "bootcmd:\n",
             "  - mkswap /dev/xvdb\n",
             "  - swapon /dev/xvdb\n",
-            "  - yum install -y aws-cli nfs-utils\n",
+            "  - until yum install -y aws-cli nfs-utils; do echo \"Waiting for network\"; done;\n",
```

Comment:
Ah I see even better now from the original PR. It will retry `yum install` until its successful